src/components/Postiterie.vue

```diff
@@ -207,16 +207,20 @@ export default {
 
     sendFile: function(){
       let $this = this;
+      
       let fileContent = '<?xml version="1.0" encoding="UTF-8"?><mxGraphModel dx="1190" dy="757" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0"><root><mxCell id="0" /><mxCell id="1" parent="0" /></root></mxGraphModel>'
-      //let fileContent = this.resultToXML()
+      //let fileContent = this.resultToXML()     // uncomment this line  and remove the one above when resultToXml is ready
       let file = new Blob([fileContent], {type: 'text/xml'});
 
-      let fileName ="";
+      let date = new Date()
+      date = date.toLocaleDateString("fr-FR");
 
-      if (this.fileName){
-        fileName = this.fileName
+      //if email is set we can store it in local storage for next time
+      if (this.email){
```

Comment:
And what happens if no email?

Reply:
I figured that receiving mail would be optional. So if you input your mail then you'll receive the mail and it will save the email for next use. But if you only use the app once it's okay to leave the field empty. What do you think about it ?

Reply:
Then it just directly opens your schema in google drive?

If that's the case, I'm fine with it!